client/sites-dashboard-v2/site-preview-pane/preview-pane-header-buttons.tsx

```diff
@@ -0,0 +1,32 @@
+import { Button } from '@automattic/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { ItemData } from 'calypso/a8c-for-agencies/components/items-dashboard/item-preview-pane/types';
+import { useSiteAdminInterfaceData } from 'calypso/state/sites/hooks';
+
+type Props = {
+	focusRef: React.RefObject< HTMLButtonElement >;
+	itemData: ItemData;
+	closeSitePreviewPane?: () => void;
+};
+
+const PreviewPaneHeaderButtons = ( { focusRef, closeSitePreviewPane, itemData }: Props ) => {
+	const { adminLabel, adminUrl } = useSiteAdminInterfaceData( itemData.blogId );
+	const { __ } = useI18n();
+	return (
+		<>
+			<Button onClick={ closeSitePreviewPane } className="item-preview__close-preview-button">
+				{ __( 'Close' ) }
+			</Button>
+			<Button
+				primary
```

Comment:
nit: hmm I don't think it needs to be fixed in this PR, but the other primary controls seem to have a darker hover state, wheras this one just keeps the same background color.

https://github.com/Automattic/wp-calypso/assets/22446385/5f12a2de-83fd-4625-8254-62ec0267a6ec

Reply:
I think this is caused due to the WP Admin button being focused when the fly-out panel is displayed. I was able to see the correct hover color once I clicked elsewhere to remove the button's focus state.

Reply:
ahhh ok I see so it was focused already, I was just thinking I should have also checked the focus state.